spec/fixtures/sign_in/logingov.crt

```diff
@@ -0,0 +1,16 @@
+-----BEGIN CERTIFICATE-----
+MIICljCCAX4CCQD6AhBNd7f/2zANBgkqhkiG9w0BAQsFADANMQswCQYDVQQGEwJV
```

Comment:
Now that this is here as a fixture, why don't we just set this path in `test.yml` settings file? Then we wouldn't even need to stub the cert/key in the test specs, it'd just pick these up

Also, could we just go ahead and set the localhost paths to these fixture files as well?